src/audio/asrc/asrc_ipc4.c

```diff
@@ -33,7 +33,6 @@
 #include <ipc4/base-config.h>
```

Comment:
It would be simpler to convert both ipc3 and ipc4 versions to module adapter. You can build and run the ipc3 version in testbench. The difference in them is very small.

Reply:
The ALSA controls handling is the biggest difference in ipc3 and ipc4 but this component does not have them.

Reply:
I also suggest this PR only focus on module converter, then do ipc3/ipc4 isolation.

Reply:
@singalsu  thanks for telling how to test IPC3 version, I will do the IPC3 version.